pyftdi/bin/pyterm.py

```diff
@@ -185,33 +300,40 @@ def _writer(self, fullmode, silent, localecho, crlf=0):
         while self._resume:
             try:
                 char = getkey()
-                if MSWIN:
-                    if ord(char) == 0x3:
-                        raise KeyboardInterrupt()
-                if fullmode and ord(char) == 0x2:  # Ctrl+B
+                if fullmode and ord(char) == 0x2:    # Ctrl+B
                     self._cleanup()
                     return
+                if MSWIN:
+                    if ord(char) in (0, 224):
+                        char = getkey()
+                        self._port.write(winfnkeys[ord(char)])
+                        continue
+                    if ord(char) == 0x3:    # Ctrl+C
+                        raise KeyboardInterrupt()
```

Comment:
I just realized this is redundant, is it better to handle here, or in the getkey() function?

Reply:
handling it here